### DISCUSSION TITLE
Make scholarly citable

### DISCUSSION THREAD

**Discussion by arunkannawadi** [scholarly-python-package/scholarly#376] [General]
While a common use case of `scholarly` is to simply update citation counts and h-indices in personal webpages, our proxy integration allows for large-scale citation network analyses in several research projects. Thus, it is useful to have a DOI for `scholarly` so that the software can be cited and those citations can be tracked. This is fairly easy to do with Zenodo, but open to other options as well.

Pinging those listed as authors in `setup.py`:  @ipeirotis @silvavn @OrganicIrradiation

**Comment by silvavn** [comment]
Agreed.

**Comment by ipeirotis** [comment]
Fine by me.

**Comment by silvavn** [comment]
I've added the repository to Zenodo, as soon as a new release is created it will be automatically collected and a DOI number will be generated. Additionally, I would say we should add a TeX entry to the `readme.md` file for easy access and a CFF file (see: https://citation-file-format.github.io/) could be added to the reposity once we have a DOI file. @arunkannawadi

**Reply by arunkannawadi** [reply]
Great, thanks @silvavn ! I will create a CFF file, but I don't quite follow what you mean by adding a TeX entry to README.

**Reply by silvavn** [reply]
A bibtex entry formatted entry as we can get from google scholar, for example.
Eg. 
```
@article{silva2017tutor,
  title={A tutor agent for moba games},
  author={Silva, Victor do Nascimento and Chaimowicz, Luiz},
  journal={arXiv preprint arXiv:1706.02832},
  year={2017}
}
```

**Comment by abubelinha** [comment]
Sorry if I opened a wrong issue https://github.com/scholarly-python-package/scholarly/issues/501 (outdated version in citation text?)
Probably this was a better place to mention that but I had not realized that scholarly has an active github discussions section

Thanks for your work!
